Sources/RemindersLibrary/CLI.swift

```diff
@@ -42,12 +42,18 @@ private struct Add: ParsableCommand {
         name: .shortAndLong,
         help: "The date the reminder is due")
     var dueDate: DateComponents?
+    
+    @Option(
+        name: .shortAndLong,
+        help: "The priority of the reminder  [0: None, 4: High, 5: Medium, 6: Low]")
```

Comment:
Where are these raw values from?

Reply:
These values are obtained by trial and error. I cannot find any documentation. [Property of EKReminder](https://developer.apple.com/documentation/eventkit/ekreminder) is simply an Int.